### ISSUE TITLE
Does model output phoneme-level timing info ?

### ISSUE THREAD

**Issue by lumpidu** [p0p4k/pflowtts_pytorch#32]
Hi, thanks for your work. I'd be interested, if the model also provides phoneme-level timing information at inference ?

**Comment by p0p4k** [comment]
Yes, the attn matrix can be used to get the frame numbers and convert to time in seconds during inference.
